Delete .gitignore in released file

I'm trying to add some personal functions to WRF-Chem.

So, I downloaded released [WRFV4.1.4](https://github.com/wrf-model/WRF/releases/tag/v4.1.4) and uploaded to my empty private repository.

Then, I cloned the repository and uploaded to the HPC.
But, I got this error when compiling WRF-Chem with KPP:
```
This is KPP-2.1.

KPP is parsing the equation file.
KPP was told to generate WRF conform code
KPP is computing Jacobian sparsity structure.
KPP is starting the code generation.
KPP is initializing the code generation.

 KPP is using the WRF conform integrator routine:
 /yin_raid/xin/tmp_lfr/WRF-4.1.4/chem/KPP/kpp/kpp-2.1/int/WRF_conform/rosenbrock

Fatal error : /yin_raid/xin/tmp_lfr/WRF-4.1.4/chem/KPP/kpp/kpp-2.1/int/WRF_conform/rosenbrock.f90: Can't read file
Program aborted
make: *** [cb05_sorg_aq_Integrator.f90] Error 3
grep: ../../../module_kpp_cb05_sorg_aq_Integr.F: No such file or directory
Makefile:50: warning: overriding commands for target `.F.o'
../../../../configure.wrf:379: warning: ignoring old commands for target `.F.o'
Makefile:58: warning: overriding commands for target `.c.o'
../../../../configure.wrf:408: warning: ignoring old commands for target `.c.o'
```

BTW, if I downloaded the official release directly on the HPC, it works well.

Actually, this issue is caused by the `*.f90` in the `.gitignore` ...
I think it's better to delete `.gitignore` in the released file.

--- a/.gitignore
+++ b/.gitignore
@@ -16,3 +16,4 @@
 configure.wrf*
 *.backup
 *.f90
+!chem/KPP/kpp/kpp-2.1/**/*.f90

--- a/.gitignore
+++ b/.gitignore
@@ -21,8 +21,11 @@ configure.wrf*
 
 ##############################################################################
 # These files are EXCLUDED from not being tracked, so they are still         #
-# tracked                                                                    #
-# The "!" character is the NOT                                               #
-# The "**" syntax is "this directory and recursively all below"              #
+# tracked (double negative logic hurts my head)                              #
+#                                                                            #
+# The leading "!" character is the NOT operator                              #
+#                                                                            #
+# The "**" syntax means "this directory level and recursively all            #
+# directories below this level"                                              #
 ##############################################################################
 !chem/KPP/kpp/kpp-2.1/**/*.f90

--- a/.gitignore
+++ b/.gitignore
@@ -17,15 +17,3 @@
 configure.wrf*
 *.backup
 *.f90
-
-
-##############################################################################
-# These files are EXCLUDED from not being tracked, so they are still         #
-# tracked (double negative logic hurts my head)                              #
-#                                                                            #
-# The leading "!" character is the NOT operator                              #
-#                                                                            #
-# The "**" syntax means "this directory level and recursively all            #
-# directories below this level"                                              #
-##############################################################################
-!chem/KPP/kpp/kpp-2.1/**/*.f90

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+##############################################################################
 # This is the top-level .gitignore file for the WRF model                    #
 #                                                                            #
 # Filenames and wildcards added below will not be tracked by git in any      #
@@ -16,4 +17,12 @@
 configure.wrf*
 *.backup
 *.f90
+
+
+##############################################################################
+# These files are EXCLUDED from not being tracked, so they are still         #
+# tracked                                                                    #
+# The "!" character is the NOT                                               #
+# The "**" syntax is "this directory and recursively all below"              #
+##############################################################################
 !chem/KPP/kpp/kpp-2.1/**/*.f90

--- a/chem/KPP/kpp/kpp-2.1/.gitignore
+++ b/chem/KPP/kpp/kpp-2.1/.gitignore
@@ -1,0 +1,9 @@
+##############################################################################
+# These files are EXCLUDED from not being tracked, so they are still tracked #
+#                                                                            #
+# The leading "!" character is the NOT operator                              #
+#                                                                            #
+# The "**" syntax means "this directory level and recursively all            #
+# directories below this level"                                              #
+##############################################################################
+!**/*.f90

